kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue

```diff
@@ -177,8 +191,37 @@
           viewport: this.getViewport(),
           pageIndex: this.pageNum - 1,
           enhanceTextSelection: true,
+          eventBus: this.eventBus,
         });
       },
+      createStructTreeLayer() {
+        this.structTreeLayer = new StrucTreeLayerBuilder(this.$refs.textLayer);
+      },
+      onTextLayerRendered(event) {
+        if (event.pageNumber !== this.pageNum) {
+          return;
+        }
+        this.eventBus.off('textlayerrendered', this.onTextLayerRendered);
+        if (!this.$refs.canvas) {
+          return; // The canvas was removed, prevent errors below.
+        }
+        // The structure tree must be generated after the text layer for the
+        // aria-owns to work.
+        this.pdfPage.getStructTree().then(tree => {
+          if (!tree) {
+            return;
+          }
+          if (!this.$refs.canvas) {
+            return;
+          }
+          const treeDom = this.structTreeLayer.render(tree);
+          treeDom.classList.add('structTree');
+          this.$refs.canvas.appendChild(treeDom);
+        });
+      },
+    },
+    $trs: {
```

Comment:
We discussed including this in the CommonCoreStrings, right @marcellamaki?
https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L150